ruby/build_jruby_release.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# This file should be executed with jruby
+
+set -ex
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
+set -euo pipefail
+if [[ ! -d "${RUNFILES_DIR:-/dev/null}" && ! -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+  if [[ -f "$0.runfiles_manifest" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles_manifest"
+  elif [[ -f "$0.runfiles/MANIFEST" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles/MANIFEST"
+  elif [[ -f "$0.runfiles/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+    export RUNFILES_DIR="$0.runfiles"
+  fi
+fi
+if [[ -f "${RUNFILES_DIR:-/dev/null}/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+  source "${RUNFILES_DIR}/bazel_tools/tools/bash/runfiles/runfiles.bash"
+elif [[ -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+    source "$(grep -m1 "^bazel_tools/tools/bash/runfiles/runfiles.bash " \
+              "$RUNFILES_MANIFEST_FILE" | cut -d ' ' -f 2-)"
+else
+echo >&2 "ERROR: cannot find @bazel_tools//tools/bash/runfiles:runfiles.bash"
+exit 1
+fi
+# --- end runfiles.bash initialization ---
+
+# Make a temporary directory and move to it to do all packaging work
+mkdir -p tmp
+cd tmp
+
+# Move all generated files to lib/google/protobuf
+mkdir -p lib/google/protobuf
+cp "$(rlocation com_google_protobuf/src/google/protobuf/any_pb.rb)" lib/google/protobuf
```

Comment:
You can use `dirname` to copy these with a wildcard

Reply:
I went back and forth on this. rlocation doesn't support wildcards, but I think you are suggesting that I get the dirname and then use wildcards which could be possible.

I ended up manually listing them as more of a precaution. Everywhere else the files are wildcarded in which I think could make it very easy to lose a file. This will fail in that case, which I think ends up being a good check. On the other hand, a new file being added could cause this to fail silently. What do you think about that tradeoff?

Reply:
If we were missing a file I would expect the gem build/test to fail in our CI.  This seems a bit redundant to me, and not worth the maintenance burden.

One alternative that might work nicely is to add an sh_test rule that actually runs the tests using the built gem.  That way we can actually run our tests locally and not have to wait for the CI to be run over a PR